src/components/recipeCard/styles.ts

```diff
@@ -0,0 +1,48 @@
+import {StyleSheet} from 'react-native';
+
+import {createThemedStyles} from '@helpers';
+import {layout} from '@theme';
+
+export const recipeCardStyles = createThemedStyles((colors, scalingMetrics) => {
+  const {wp, scaleSize, moderateScale} = scalingMetrics;
+  return StyleSheet.create({
+    cardContainer: {
+      height: wp(60),
+      marginHorizontal: scaleSize(10),
+      backgroundColor: colors.gray50,
+      borderRadius: scaleSize(10),
+      width: wp(60),
+      elevation: 8,
+      shadowOpacity: 0.5,
+      shadowRadius: 12,
+      shadowColor: colors.gray800,
+      shadowOffset: {width: scaleSize(8), height: scaleSize(8)},
+    },
+    image: {
+      flex: 2,
+      width: '100%',
+      borderTopLeftRadius: scaleSize(10),
+      borderTopRightRadius: scaleSize(10),
+    },
+    bottomContainer: {
+      flex: 1,
+      paddingHorizontal: scaleSize(10),
+    },
+    title: {
+      fontSize: moderateScale(16),
```

Comment:
are you using default font in app??

Reply:
Yes, just for time being. I might add custom fonts later.

Reply:
You are required to add custom fonts if you are planning to do it later, it will need extra time make sure to manage it well